### PR TITLE
Reduce batch sizes

### PR DESCRIPTION
**Pull request by melvinfolkers**
In order to reduce batch sizes of modules 100 and 200, i have added a second (optional) filter to the batchlist. The batch list is now a list of nested dictionaries. 

Each list element (batch) has a `period` key and a `amount` key.  These two keys have in their turn two keys: `from` and `to`.

In the script there is a check if the amount key is set. This will only be set in the function periods_from_start which is used by module 100 and 200. By adding this amount filter, the batches are halved in size.

This should hopefully fix the issue of the Twinfield server timing out when there is a large request.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,4 +8,17 @@
 
 ---
 
-- Set mutations query to periods of months.
\ No newline at end of file
+- Set mutations query to periods of months.
+
+# 1.0.8
```
Comment (melvinfolkers) [inline CHANGELOG.md:13]: @erfannariman  de changelog is per ongeluk in de commit van de "fixes for black" meegegaan. Wel even belangrijk dat je deze ziet. Ik had namelijk het idee dat we inmiddels op versie 1.0.9 zaten

Comment (erfannariman) [reply CHANGELOG.md:13]: Ja dat klopt, was voor de requirements.txt. Dus wat je gedaan hebt klopt.

**Comment by erfannariman** [comment]
@melvinfolkers had je nog wat in `ci_audit.sh gezet`? Want deze is nu leeg.

**Review by erfannariman — COMMENTED** [review]
Ziet er goed uit! Nice

**Pull request merged**